pyvista/plotting/errors.py

```diff
@@ -0,0 +1,30 @@
+"""Core and plotting errors."""
```

Comment:
Only Plotting errors should be here :)
```suggestion
"""Plotting errors."""
```